python/pyspark/sql/streaming/StateMessage_pb2.pyi

```diff
@@ -0,0 +1,139 @@
+from google.protobuf.internal import enum_type_wrapper as _enum_type_wrapper
```

Comment:
Do we need to commit generated protobuf files each time, or can they be generated at build time?

Reply:
I think we have to commit it each time, we were doing the same for Spark Connect generated proto files